### PR TITLE
printSet util method

### PR DESCRIPTION
**Pull request by literallyaddi**

### CODE REVIEW

**Comment by addipls** [comment]
image

File: Extreme_Motorcycle_Class/src/emc/Utils.java [hunk 1]
```diff
@@ -10,4 +15,69 @@ public static LocalDate createLocalDate(int year, int month, int day) throws Dat
...
+        // Do not modify the loop values: the result sets column index from 1 while a List index starts at 0.  The plus ones are to compensate for this.
+        for (int i = 0; i < columnCount; i++) {
+            try {
+                columnNames.add(rsmd.getColumnName(i + 1));
+            } catch (SQLException e) {
+                System.out.println("Error getting column name");
+                e.printStackTrace();
+                return;
+            }
+        }
```
Comment (reharri7) [inline Extreme_Motorcycle_Class/src/emc/Utils.java:51]: https://stackoverflow.com/a/10213258/14839305 Maybe consider something like this. May not be a good idea without making objects and may be over-engineered for our purposes

Comment (addipls) [reply Extreme_Motorcycle_Class/src/emc/Utils.java:49]: Not very concerned with efficiency for the scope of this project, just need to resolve issues displaying the table in a properly formatted way for all cases
image

**Comment by addipls** [comment]
This will be included when I merge the "addi" branch instead

**Pull request closed without merging**